src/kent/app.py

```diff
@@ -235,80 +256,138 @@ def store_view(project_id):
         else:
             body = request.data
 
-        # If it's JSON, then decode it
+        app.logger.debug(f"{body}")
+
+        # JSON decode it
         try:
             json_body = json.loads(body)
         except Exception:
-            app.logger.exception("%s: exception when JSON-decoding body.", error_id)
-            app.logger.error("%s: %s", error_id, json_body)
+            app.logger.exception("%s: exception when JSON-decoding body.", event_id)
+            app.logger.error("%s: %s", event_id, json_body)
             body = {"error": "Kent could not decode body; see logs"}
             raise
 
-        ERRORS.add_error(error_id=error_id, project_id=project_id, payload=json_body)
+        EVENTS.add_event(event_id=event_id, project_id=project_id, body=json_body)
 
         # Log interesting bits in the body
         if "exception" in json_body:
             app.logger.info(
                 "%s: exception: %s %s",
-                error_id,
-                deep_get("exception.values.[0].type", json_body),
-                deep_get("exception.values.[0].value", json_body),
+                event_id,
+                deep_get(json_body, "exception.values.[0].type"),
+                deep_get(json_body, "exception.values.[0].value"),
             )
         if "message" in json_body:
-            app.logger.info("%s: message: %s", error_id, deep_get("message", json_body))
+            app.logger.info("%s: message: %s", event_id, deep_get(json_body, "message"))
         app.logger.info(
             "%s: sdk: %s %s",
-            error_id,
-            deep_get("sdk.name", json_body),
-            deep_get("sdk.version", json_body),
+            event_id,
+            deep_get(json_body, "sdk.name"),
+            deep_get(json_body, "sdk.version"),
         )
 
         # Log event url
-        event_url = f"{request.scheme}://{request.headers['host']}/api/error/{error_id}"
-        app.logger.info("%s: project id: %s", error_id, project_id)
-        app.logger.info("%s: url: %s", error_id, event_url)
+        event_url = f"{request.scheme}://{request.headers['host']}/api/event/{event_id}"
+        app.logger.info("%s: project id: %s", event_id, project_id)
+        app.logger.info("%s: url: %s", event_id, event_url)
+
+        return {"success": True}
+
+    @app.route("/api/<int:project_id>/envelope/", methods=["POST"])
+    def envelope_view(project_id):
+        app.logger.info(f"POST /api/{project_id}/envelope/")
+        request_id = str(uuid.uuid4())
```

Comment:
We don't actually use the request_id anywhere. I'm not sure what to do with this at this point. I figure I'll get a better understanding once I see envelopes with more than one item.